build/build.xml

```diff
@@ -48,6 +49,12 @@ External Dependencies:
 	<echo>Loading base properties</echo>
 	<loadproperties srcFile="build.properties"/>
 
+	<!-- Load secret properties -->

```

Comment:
I was getting an error when trying to run a full build because it couldn't find the secret properties file... the rearranging of the properties makes it so folks can still run a full build, even if they aren't us-- it just won't be signed.